contents/newsletter/how-to-get-job-startup.md

```diff
@@ -0,0 +1,111 @@
+
+* A genuine and deep knowledge of the product you’ll be working on.
+* Do a little something to make you stand out. For example, as part of his application, Marcus, one of our support engineers, [opened a PR](https://github.com/PostHog/posthog.com/pull/4922) to add him to the team page. It certainly got our attention.
+
+![marcus application](https://res.cloudinary.com/dmukukwp6/image/upload/marcushof_2dbd558b21.png)
+
+### One simple, annoying trick
+
+Get ready to groan, but there’s one thing that will really help: get a proper, warm referral. We trust our team’s judgment, so a referral can really help us pre-filter applicants. 
+
+This does not mean you should randomly spam people on LinkedIn for an intro, but it does mean you should work hard wherever you go and leave a good impression. People talk to each other. And, who knows, one day maybe they’ll talk about you and be really nice.
+
+## Ask the right questions
+
+One way we’re able to recognize [10x engineers](https://newsletter.posthog.com/p/beyond-the-10x-engineer) at the interview stage is by the questions they ask. It’s not rude to ask very specific questions, it shows you’re genuinely curious about what we’re doing. These are the questions you should be asking when interviewing with a startup founder.
+
+### Does the company have product-market fit?
+
+Very few engineers ask this when we do interviews. Ask! You can and should also ask about revenue and the number of daily active users. Very early-stage startups might not have product-market fit, but they should have a plan for how to get there quickly.
+
+[Strong product-market fit](https://posthog.com/founders/measure-product-market-fit) happens when a company is able to combine significant top-line growth with meaningful usage and a flat retention curve.
+
+### How much runway does the company have?
+
+Or, simply put: is the company default alive? A company with very little runway and no product-market fit is likely to have a challenging culture. Asking these kinds of questions will get you much closer to understanding what the company is like to work for than lazily asking about the company’s culture.
+
+Sure, a startup might not be profitable – but it should have a plan to be profitable before it runs out of money. If it doesn’t, ask why. Avoid companies that don’t know if they’re default alive or not – or don’t care about it. 
+
+### If you could throw money at a problem and solve it immediately, what would it be?
+
+Superstar engineers are laser-focused on working out where a company is struggling and how they can help. Asking this question shows you want to understand our biggest problems and are ready to throw yourself into solving them by shipping and shipping fast. 
+
+If you get the role, you need to be proactive in speaking to users, understanding problems big and small, taking ownership of them, and fixing them.
+
+## Know the signs of a great engineering role
```

Comment:
I feel like this is interesting but not really relevant to getting a great engineering role. If I put myself in the shoes of a junior engineer trying to get a job, I'm applying to any place that has an opening. 

If I'm a really senior engineer, a referral/recruitment + brand + finances matter much more.

Reply:
i think it's good to have this in the mix of points, and is something we're especially strong on so worth making some noise about. but agree we want to big up the referral point and some other points higher up. will tweak elsewhere but suggest we keep this as is

Reply:
I think the idea is right, it just needs be less "this is what makes a good ad" and more "here's how to pick a winning company".

- A bit like with the cover letters, it would be useful if we could have some side-by-side of a good ad versus a bad one, just to make what we're talking about more real.

- I think we could add some nice advice around recognising signs of traction, such as:
  - Researching mentions of the company on social. Are they seeing spontaneous word-of-mouth?
  - Check Google Trends to see if there's a growing trend of searches for their brand.
  - Who are the investors? What's their track record?

- Does the funding match the volume of hiring? - i.e. are they over hiring, hiring lots of sales people despite limited funding?

- Background of founders. Are they first-timers, or have previous exits? Are they "serial" founders who haven't found any success.

There's more we could write, but I think this + what we already have would add a lot more actionable / tactical advice people can follow.